### PR TITLE
Convert to TS

### PR DESCRIPTION
**Pull request by zautumnz**
## Change Type

* [ ] Feature
* [x] Chore
* [ ] Bug Fix

## Change Level

* [x] major
* [ ] minor
* [ ] patch

## Change Description

This will be a breaking change, because no more Flow.

## Checklist

* [x] Followed PR template
* [ ] Added unit tests if applicable
* [ ] Added contract tests if applicable
* [ ] Added integration tests if applicable
* [ ] Added e2e tests if applicable

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10
+    working_directory: ~/repo
```
Comment (zacanger) [inline .circleci/config.yml]: CI!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -13,11 +13,3 @@
 ## Change Description
 
 #### Relevant Links (Task, Bug Report, etc.)
-
-## Checklist
```
Comment (zacanger) [inline .github/PULL_REQUEST_TEMPLATE.md]: This stuff doesn't make much sense on this project

File: README.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # react-dwolla-iav
 
+![npm version](https://img.shields.io/npm/v/react-dwolla-iav.svg)](https://npm.im/react-dwolla-iav) [![CircleCI](https://circleci.com/gh/jane/react-dwolla-iav.svg?style=svg)](https://circleci.com/gh/jane/react-dwolla-iav)
```
Comment (zacanger) [inline README.md]: Badges!

File: README.md [hunk 4]
```diff
@@ -10,8 +12,7 @@ Dwolla doesn't provide a library for their `Dwolla.js` client-side scripts. This
 component wraps their script and provides a React component for embedding their
 IAV flow.
 
-* [IAV demo](https://iavdemo.dwolla.com/)
```
Comment (zacanger) [inline README.md]: I removed this link because that site has been down for a while.

File: src/index.tsx [hunk 5]
```diff
@@ -20,22 +18,22 @@ export const load = (environment: 'prod' | 'sandbox'): Promise<void> =>
...
+
       document.body.appendChild(s)
     }
   )
 
 type DwollaIAVError = {
-  code: string,
-  message: string,
+  code: string
+  message: string
```
Comment (jhelst) [inline src/index.tsx]: no commas here in TS?

**Review by jhelst — APPROVED** [review]

File: src/index.tsx (hunk 5 above)
Comment (zacanger) [reply src/index.tsx]: I guess not, Prettier did this.

Comment (TomPridham) [reply src/index.tsx]: no. the options are semi colons or nothing. it's weird.

**Pull request merged**